cmd/kubebuilder/docs/docs.go

```diff
@@ -99,17 +108,23 @@ func RunDocs(cmd *cobra.Command, args []string) {
 	}
 
 	if generateConfig {
+		// Regenerate the config.yaml with the table of contents
+		os.Remove(filepath.Join(outputDir, "config.yaml"))
 		CodeGenerator{}.Execute(wd)
 	}
 
-	// Run the docker command to build the docs
-	c := exec.Command("docker", "run",
-		"-v", fmt.Sprintf("%s:%s", filepath.Join(wd), "/host/repo"),
-		"-e", "DOMAIN="+util.GetDomain(),
-		"-e", "DIR="+filepath.Join("src", util.Repo),
-		"-e", "OUTPUT="+outputDir,
-		"gcr.io/kubebuilder/gendocs:alpha4",
-	)
+	// Make sure to generate the openapi
+	generatecmd.Codegenerators = []string{"openapi"}
+	generatecmd.RunGenerate(cmd, args)
+
+	// Create the go program to create the swagger.json by serializing the openapi go struct
+	cr := util.GetCopyright(copyright)
+	doSwagger(wd, swaggerTemplateArgs{
```

Comment:
May be rename it to doSwaggerGenerator ? because it is generating a generator Go program not the swagger itself ?